app/src/main/res/layout/audio_fragment.xml

```diff
@@ -54,27 +54,24 @@
       android:layout_marginStart="16dp"
       android:thumbOffset="16dp"
       app:layout_constraintBottom_toBottomOf="parent"
-      app:layout_constraintEnd_toStartOf="@+id/tvAudioLanguage"
+      app:layout_constraintEnd_toStartOf="@+id/ivAudioLanguage"
       app:layout_constraintStart_toEndOf="@+id/ivPlayPauseAudio"
       app:layout_constraintTop_toTopOf="parent" />
 
-    <TextView
-      android:id="@+id/tvAudioLanguage"
+    <ImageView
```

Comment:
apply reformat and rearrange code from the menu in Android Studio.

Reply:
done